### PR TITLE
chore: deprecate pictograms

### LINKED ISSUE #6923: Determine changes in pictogram assets
The brand team has delivered a set of new pictogram assets that have been redrawn to support `fill` instead of `stroke`. We need to determine the scope of these changes in order to verify the following:

- What pictogram assets have changed? (For example, what has been renamed)
- What pictogram assets have been removed? What are they to be replaced with? (if anything)
- What pictogram assets are staying the same?

[Pictogram_Breaking_Change.zip](https://github.com/carbon-design-system/carbon/files/5293550/Pictogram_Breaking_Change.zip)

[Master_naming_update_091020.xlsx](https://github.com/carbon-design-system/carbon/files/5293566/Master_naming_update_091020.xlsx)

Using this information, we can coordinate a migration path for the affected assets when we release our breaking change.

### PR DESCRIPTION
**Pull request by emyarod**
Closes #6923

This PR removes all of the pictograms which were marked for deprecation

#### Changelog

**Changed**

- pictogram metadata

**Removed**

- deprecated pictograms

### CODE REVIEW

**Comment by joshblack** [comment]
@emyarod how are we anticipating folks using the deprecation warnings to move to the correct asset?

It seemed like in the PR that was merged that we added new assets and the deprecation warnings, but I'm not sure when those deprecation warnings will get seen by an end-user.

Is the hope that what is in master will get published and this change gets merged in? Or is the hope to merge this in and do something like a migration guide? Just curious 👀

**Comment by emyarod** [comment]
we assumed since the pictogram updates are already considered breaking changes, the aliases in the new/updated pictogram metadata would point users to the right assets. Should we just be deprecating pictograms that weren't touched in #7094? The migration process is not really clear to us especially since #7094 is an irregular update

**Comment by joshblack** [comment]
@emyarod I think removing the deprecated assets in the major change makes a lot of sense, I was just wondering how we were communicating these changes to users and helping them migrate from old to new. It seems like it is unrealistic for the deprecation messages to be included in a v10 release because the new assets wouldn't be available. On the other hand, including the deprecated assets with the message in v11 would just mean holding on to them for longer.

In this case, it could just be that there is a migration guide needed to help folks know what the changes are to help them update. We could also do a codemod as well that uses that list to automatically update codebases that use these assets.

Not sure what the right answer is here, just am trying to figure out what the migration experience will look like for teams.

**Comment by emyarod** [comment]
I'm not familiar with the process when it comes to asset updates like this as typically @dudley-ibm handles these IIRC. I'm also not sure what the right answer is but I think it makes sense to involve brand here and how they want to communicate these changes

**Comment by andreancardona** [comment]
The migration guide for Pictograms has been added to this PR 🎉

File: docs/migration/10.x-pictograms.md [hunk 1]
```diff
@@ -0,0 +1,175 @@
+# Pictograms
```
Comment (joshblack) [inline docs/migration/10.x-pictograms.md]: Would this be 11.x-pictograms.md? 👀

**Review by joshblack — COMMENTED** [review]
Looking great! Should we have a guide for pictograms-react as well with the name changes?

File: docs/migration/10.x-pictograms.md (hunk 1 above)
Comment (andreancardona) [reply docs/migration/10.x-pictograms.md]: yes looks like @emyarod updated to `11.x-pictograms.md`!

Comment (andreancardona) [reply docs/migration/10.x-pictograms.md]: @joshblack regarding a migration guide for Pictograms React - it looks like didn't do that for icons? 👀

Comment (joshblack) [reply docs/migration/10.x-pictograms.md]: @andreancardona IIRC the icons migration guide was specifically for `carbon-icons`. I think `@carbon/icons-react` was published with v10 and so was using the new names by default 👍

**Comment by mjabbink** [comment]
Looking good from what I can tell. I only see one pictogram missing (Folder) from this preview
https://deploy-preview-7272--carbon-elements.netlify.app/pictograms/examples/preview/

File: docs/migration/10.x-pictograms.md (hunk 1 above)
Comment (andreancardona) [reply docs/migration/10.x-pictograms.md]: done!

**Comment by emyarod** [comment]
the file is still present in the repo but this may just be a bug with the pictogram preview

**Comment by andreancardona** [comment]
@joshblack just added the migration guide for `Pictograms-React`

**Comment by joshblack** [comment]
I believe the folder asset in the preview is using stroke still which is why it is not displaying 🔍

**Comment by emyarod** [comment]
cc @dudley-ibm do you have an updated `Folder` pictogram that does not use stroke? The asset we have on hand still uses stroke

**Comment by dudley-ibm** [comment]
[folder.svg.zip](https://github.com/carbon-design-system/carbon/files/5569660/folder.svg.zip)
Sent it in Slack, but here it is also.

**Comment by andreancardona** [comment]
@joshblack @mjabbink `Folder.svg` has been updated

Thankssss @emyarod 🥳

**Comment by emyarod** [comment]
the asset has been updated and is now visible in the pictograms preview

**Review by joshblack — APPROVED** [review]
Looks great! For the commit, would it make sense to do something like:

```git
feat(pictograms): remove deprecated assets

BREAKING CHANGE: deprecated assets have been removed
```

For conventional commits?

**Comment by dudley-ibm** [comment]
approved

**Comment by emyarod** [comment]
@joshblack makes sense, how would we implement that when the commits will be squashed when we merge?

**Comment by joshblack** [comment]
@emyarod I think you can edit the merge commit when merging to match that format 👀

**Comment by mjabbink** [comment]
@conradennis we only need one Prague right? Also only need one Weather

Screen Shot 2020-11-19 at 5 03 51 PM


Screen Shot 2020-11-19 at 5 06 23 PM

**Comment by emyarod** [comment]
@joshblack got it, will make that change when merging. as for the asset comments, we will need to wait for them to be resolved before merging this PR (and merging into v10.25)

**Comment by conradennis** [comment]
@mjabbink yup correct, the first one should be removed.

**Comment by andreancardona** [comment]
> @mjabbink yup correct, the first one should be removed.

@conradennis the first one should be removed for `prague` and `weather`? Please clarify 👀 also please specify which code names.

**Comment by dudley-ibm** [comment]
The artwork for the `weather` pictogram initially submitted in the Industry category was similar but slightly different than the `weather` pictogram submitted afterward in the Weather category. That's why I added the 01 & 02, but they're definitely close enough that we only need one.

**Comment by dudley-ibm** [comment]
@emyarod @andreancardona it looks like I did not communicate that `prague.svg` should be deprecated in favor of `prague--charles-bridge-tower.svg`. Can that be updated before this merges tomorrow?

**Comment by emyarod** [comment]
`prague.svg` has been removed

**Comment by mjabbink** [comment]
👍

**Comment by dudley-ibm** [comment]
Since `weather--01.svg` and `weather--02.svg` are so similar in artwork, let's kill the one in this PR slated for the Industry category (weather--01.svg) and only update the svg that appears in the Weather category. It will now just be called `weather.svg` again, and that file is attached.

[weather.svg.zip](https://github.com/carbon-design-system/carbon/files/5574447/weather.svg.zip)

**Comment by emyarod** [comment]
`weather--01` and `weather--02` have been replaced by `weather`

**Review by andreancardona — APPROVED** [review]

**Comment by mjabbink** [comment]
Go to the glass! Woohoo!

**Pull request merged**